### PR TITLE
feature(python): Python info and Recommendations

### PR DESCRIPTION
**Pull request by michaelachrisco**
## Changes
1. Added Basic Description, Philosophy, Learning, Recommendations and Links to python page
2. Added known good resources for Python

## Purpose
For over 2 years, we have kept #131 open and debated. I want to at least provide some basic documentation around Python for new developers as well as  seasoned developers. Discussion is welcome!

## Approach
Provides a launching point that contains some known good resources for Python the language. 

## Learning
Most of the learning came from the original issue #131. We had a lot of good discussion around best practices and Code Quality that sadly didn't make it into the repo. 
 

I'm making this specifically about **python and not Django**. Django will be another separate PR and has its own dedicated page. 

References #131

### CODE REVIEW

**Review by lizl714 — APPROVED** [review]
looks good to me.  the philosophy made me laugh in a good way

File: best-practices/server-side/framework/python/README.md [hunk 1]
```diff
@@ -1,5 +1,47 @@
...
+
+## Links and Services
+* In the Fresno area? Why not try the [Fresno Python User Group](https://fresnopython.com)? 
+* Codewars [Python excercises](https://www.codewars.com/collections/basic-python).
+
 ## Frameworks for python
 
-- [Django](django/README.md)
\ No newline at end of file
+- [Django](django/README.md)
```
Comment (amerikan) [inline best-practices/server-side/framework/python/README.md:47]: Flask is also popular.

Comment (michaelachrisco) [reply best-practices/server-side/framework/python/README.md:47]: @amerikan Good suggestion. Maybe we should add a GH Issue/page for it? Do we have any products that use flask that we can reference?

We have the framework referenced here: https://github.com/Shift3/standards-and-practices/blob/main/best-practices/server-side/framework/README.md but not current pages or docs.

**Review by AramayisO — APPROVED** [review]
Looks good!

One other good (I believe "best") practice I can think of is to use a virtual environment to isolate project dependencies. Personally, I prefer setting up a virtual environment with the [built-in venv module](https://docs.python.org/3/tutorial/venv.html), but I know `pipenv` and `virtualenv` are also popular options in the python community.

**Comment by michaelachrisco** [comment]
> Looks good!
> 
> One other good (I believe "best") practice I can think of is to use a virtual environment to isolate project dependencies. Personally, I prefer setting up a virtual environment with the [built-in venv module](https://docs.python.org/3/tutorial/venv.html), but I know `pipenv` and `virtualenv` are also popular options in the python community.

Thanks! Yeah I didn't add virtual environments because I knew it would be opening up a can of worms :) Im personally on team `virtualenv` side but I knew a lot of developers on the `pipenv` side. 

Tempted to let this be up to the developer running the project but I know each tool has their particularities...so im unsure which is "best" for the shop at this time. What do you think? Should we add a section on virtual environments and jsut mention both? I suppose I could put it up to the slack channel as well.

File: best-practices/server-side/framework/python/README.md (hunk 1 above)
Comment (michaelachrisco) [reply best-practices/server-side/framework/python/README.md:47]: Added reference via: 9454ae2696493cf2c0e166304f1196e683b003bc

**Pull request merged**